cmd/helm/install.go

```diff
@@ -181,6 +182,10 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 		return nil, err
 	}
 
+	if chartRequested.Metadata.Deprecated {
+		fmt.Fprintln(out, "WARNING: Current chart is deprecated")
```

Comment:
Current chart is deprecated --> This chart is deprecated

Reply:
Fixed